_includes/head.html

```diff
@@ -4,11 +4,11 @@
 <head>
   {% include force-https.html %}
   <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
you need to keep this content variables.

may be you accidentally removed them.

add back if there's no problem

Reply:
yeah I will look into it